stylesheets/_component.timeline.scss

```diff
@@ -0,0 +1,689 @@
+                animation-name: newContentAfter;
+                -webkit-transform-style: preserve-3d;
+            }
+        }
+
+        .timeline__item--highlighted & {
+            -webkit-animation-duration: 7s;
+            animation-duration: 7s;
+            -webkit-animation-name: highlightedContentBackgroundAndBorderFade;
+            animation-name: highlightedContentBackgroundAndBorderFade;
+
+            &::before {
+                -webkit-animation-duration: 7s;
+                animation-duration: 7s;
+                -webkit-animation-name: highlightedContentBefore;
+                animation-name: highlightedContentBefore;
+                -webkit-transform-style: preserve-3d;
+            }
+
+            &::after {
+                -webkit-animation-duration: 7s;
+                animation-duration: 7s;
+                -webkit-animation-name: highlightedContentAfter;
+                animation-name: highlightedContentAfter;
+                -webkit-transform-style: preserve-3d;
+            }
+        }
+    }
+
+    &__title {
+        color: #303e49;
+        display: inline-block;
+        margin: 0 0 0 26px;
+
+        @include respond-min($screen-phone) {
+            margin: 0 0 0 32px;
+        }
+
+        @include respond-min($screen-tablet) {
+            line-height: $line-height-base;
+            margin: 0;
+            width: 60%;
+        }
+    }
+
+    &__note-title {
+        margin: 1em 0;
+    }
+
+    &__text {
+        @include word-wrap();
+
+        margin: .5em 0;
+
+        @include respond-min($screen-tablet) {
+            margin: 1em 0;
+        }
+
+        a {
+            word-break: break-all;
+        }
+
+        .avatar {
+            background-color: color(white);
+            vertical-align: bottom;
+        }
+    }
+
+    &__small {
+        color: color(gray, dark);
+        display: inline-block;
+        font-size: .7em;
+
+        .label {
+            font-size: 100%;
+        }
+    }
+
+    &__date {
+        display: block;
+        font-size: 14px;
+        margin-top: $padding-base;
+
+        @include respond-min($screen-tablet) {
+            display: inline-block;
+            float: right;
+            margin-top: 0;
+            text-align: right;
+            width: 40%;
+        }
+    }
+
+    &__attachments {
+        margin: 1em 0;
+
+        li {
+            display: inline-block;
+            margin-right: .5em;
+            width: 150px;
+        }
+    }
+
+    &__thumbnail-link {
+        border-radius: 4px;
+        cursor: pointer;
+        display: inline-block;
+        max-height: 250px;
+        max-width: 100%;
+        overflow: hidden;
+        position: relative;
+
+        @include ie-lte(10) {
+            max-width: 150px;
+        }
+    }
+
+    &__thumbnail {
+        max-height: 250px;
+        max-width: 100%;
+
+        @include ie-lte(10) {
+            max-width: 150px;
+        }
+    }
+
+    &__changes {
```

Comment:
This is specific to CXM timeline "diffs" where we show data changing from one value to another, does it need to be in Pulsar? Same with all the nested and diff selectors.